.github/workflows/p2p-workflow-fastfeedback.yaml

```diff
@@ -96,6 +95,23 @@ jobs:
       dry-run: ${{ inputs.dry-run }}
       working-directory: ${{ inputs.working-directory }}
 
+  integration-test:
+    needs: [nft-test]
```

Comment:
In the [comment](https://github.com/coreeng/project-platform-accelerator/issues/394#issuecomment-2314924061), you say that expected sequence is func -> integration -> nft.
If I understand it correctly, this `needs` makes the sequence: func -> functiona-test -> integration

In addition, I think we should add in `promote.needs` `integration-test` target. Otherwise it can promote without the test